electron/main.ts

```diff
@@ -179,7 +179,8 @@ async function createWindow(): Promise<BrowserWindow> {
     handleExit(mainWindow)
   })
 
-  if (!app.isPackaged) {
+  const isDevEnvSet = (process.env?.ELECTRON_IS_DEV ?? '0') === '1'
```

Comment:
Maybe we could have a default env variable with the dev env disabled but we could enable it using an env var when calling Heroic, so we could enable the DEV env with a packaged app as well in case we want to ask someone to debug. Makes sense?

So we could remove the `isPackaged` and defined the var to be 0 by default. On the npm script `electron` and `start` we could pass it as 1. Then, after we package the app we can call:
` ELECTRON_IS_DEV=1 heroic` and have access to more debugging like the react dev tools and any other we want.

Maybe I am overthinking this 😅

Reply:
On the contrary, I was thinking about that too and it doesn't seem to be a bad idea, so I'll do it.